student.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(classroom = 'Unknown')
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] same as in teacher, kindly update the arguments in the student initialize to match the argument passed while creating student

Reply:
Thank you so much i have done this